layouts/_partials/join-form.html

```diff
@@ -3,7 +3,7 @@
   name="subscribe"
   class="ml-0 mt-4 max-w-2xl rounded-lg bg-red-600"
   data-netlify="true"
-  action="{{ relLangURL "join-success" }}"
+  action="{{ relLangURL "join/success" }}"
```

Comment:
Now a subpath :rocket:

Can also be tested on the preview deploy by going to /nl/join/success. Also, the path shows up correctly in the /nl/join page (inspect the form)